imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml

```diff
@@ -124,6 +124,9 @@ post:
 
         **AML_CHECK_FAILED**
         AML check failed. This cannot be resolved.
+
+        **UNSUPPORTED_KYC_REGION**
+        The KYC region is not yet supported.
```

Comment:
the idea was to link to the region guide here, but because the guide is on hold, will do it later